microgpt.py

```diff
@@ -48,6 +48,7 @@
     memory = get_memory_instance()
     context = objective
     thought = "I awakened moments ago."
+    memory.add(f"{objective}, {thought}")
```

Comment:
I believe this doesn't make sense. The thought will be added to memory later on anyway and the objective is repeated with every prompt.

Reply:
Ah, chromadb complains on an empty index. You're right maybe there's a better way

Reply:
Can you figure this one out?

Reply:
I guess I can return the same context if the index is empty? Like...

```
        def get_context(self, data, num=5):
            index_max = self.index.count()

            if index_max == 0:               <--------
                return data                  <--------
            elif index_max < num:
                num = index_max

            results = self.index.query(
                query_texts=[data],
                n_results=num
            )

            results_list = results["documents"][0]
            context = "\n".join(results_list)
            context = self.summarize_memory_if_large(
                context, self.max_context_size)

            return context
 ```